relayers.json

```diff
@@ -21,19 +21,38 @@
             }
         ]
     },
-    {
+    "e8d27d8d-ddf6-48b1-9663-60b0a3ddc716": {
         "name": "ERC dEX",
         "homepage_url": "https://ercdex.com",
         "app_url": "https://app.ercdex.com/",
-        "header_img" : "ercdex.png",
+        "header_img": "ercdex.png",
         "logo_img": "ercdex.png",
         "networks": [
             {
                 "networkId": 1,
                 "sra_http_endpoint": "https://app.ercdex.com/api/v2/",
                 "sra_ws_endpoint": "wss://app.ercdex.com/ws/v2",
                 "static_order_fields": {
-                    "fee_recipient_addresses": ["0x58a5959a6c528c5d5e03f7b9e5102350e24005f1","0x7df569a165bee41ca74374c76bd888ea02dcc4a8","0xa71deef328b2c40d0e6525cd248ae749e9208dbb","0x3d974ce554fec1acd8d034f13b6640b300689a37","0xbd069e7ad0b7366ed1f0559dd8fe3e8efc0c4a72","0x4411c446756f8ed22343e8fbe8d24607027daffd","0xee2d43b8e4b57477acc2f4def265fe2887865ac0","0x5bf2c11b0aa0c752c6de6fed48dd56fed2a4286d","0x1dd43bbe2264234bccfbb88aadbde331d87719ee","0x8bf0785306eb675e38b742f59a7fcf05fccdf2b7","0x1956f5afa5d21000145e6cd2fa8ce3f52fa40875","0xa5b8d094f8364a9771c7a2287ee13efa08f847a4","0xc95bf3d3b4d6619119f3a8e29ec1d73ee801b9df","0x28f5cf7044f509af67f473c18b1f5f4f97fb4ce9","0x3b4ce2ea700ff327c3b4fe624f328c4106fd2885","0x3fa5f23d42847e49d242496ffe2a3c8fda66706c","0xd592cfa56f4c443fb27008329d67ed7d4edb59c0","0x173a2467cece1f752eb8416e337d0f0b58cad795"]
+                    "fee_recipient_addresses": [
```

Comment:
Random aside: do any linting / pretty printing in the yarn test / build? Might be a nice wishlist item if not.

Reply:
Not currently but we could easily add it.